graphql/utils/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const PHONE_CODE_EXPIRY = 2
+export const TOKEN_EXPIRY = 14
```

Comment:
we should have a comment here on each constant with the unit they represent, minutes, hours, days. Also, semicolons are missing.